optaplanner-quickstarts/quarkus-school-timetabling/pom.xml

```diff
@@ -25,6 +25,8 @@
     <deploy-plugin.version>2.8.2</deploy-plugin.version>
     <surefire-plugin.version>2.22.2</surefire-plugin.version>
     <docker-plugin.version>0.28.0</docker-plugin.version>
+    <!-- TODO Workaround for "mvn quarkus:dev" for https://github.com/quarkusio/quarkus/issues/12715 -->
+    <noDeps>true</noDeps>
```

Comment:
Probably should have been in a separate PR, as it's unrelated, but let's just merge it along.
 Anyway, took me 2 weeks to figure out this one line workaround fix (master doesn't suffer from it).